src/Core/Grid/Query/AliasQueryBuilder.php

```diff
@@ -117,7 +117,7 @@ private function applyFilters(QueryBuilder $builder, SearchCriteriaInterface $se
             }
 
             $builder
-                ->andWhere($allowedFiltersMap[$filterName] . ' LIKE :' . $filterName)
+                ->andWhere($allowedFiltersMap[$filterName] . ' = :' . $filterName)
```

Comment:
nope.
Only id_alias and active fields should be strict.
"alias" and "search" should stay with wildcard.